### DISCUSSION TITLE
Tips for increasing performance ??

### DISCUSSION THREAD

**Discussion by ghost** [pmndrs/cannon-es#141] [Q&A]

**Comment by marcofugaro** [comment]
Please elaborate your question.

Also if you're interested in performances, check out the worker example https://pmndrs.github.io/cannon-es/examples/worker or the worker + SharedArrayBuffer example https://pmndrs.github.io/cannon-es/examples/worker_sharedarraybuffer

**Comment by ghost** [comment]
@marcofugaro https://github.com/calypsu/Three.js-project/blob/main/src/index.js, what can I improve here for performance ???

**Comment by marcofugaro** [comment]
@Padmanabh82 do you have a live link?

**Reply by ghost** [reply]
Yes, https://github.com/calypsu/Three.js-project/blob/main/src/index.js

**Comment by marcofugaro** [comment]
@Padmanabh82 please put your code on a codesandbox

**Reply by ghost** [reply]
https://codesandbox.io/embed/sleepy-monad-e39fo?codemirror=1

**Reply by ghost** [reply]
sorry plese dont use that

**Reply by ghost** [reply]
my latest commit is not getting accepted by code sand box dont know why
